spec/index.bs

```diff
@@ -1678,23 +1793,19 @@ To get a {{GPUAdapter}}, use {{GPU/requestAdapter()}}.
 <script type=idl>
 [Exposed=(Window, DedicatedWorker), SecureContext]
 interface GPUAdapter {
-    readonly attribute DOMString name;
     [SameObject] readonly attribute GPUSupportedFeatures features;
     [SameObject] readonly attribute GPUSupportedLimits limits;
     readonly attribute boolean isFallbackAdapter;
+    readonly attribute GPUAdapterInfo info;
```

Comment:
Since this returns a new object each time and the value can change due to unmasking, maybe this should be a method?

Reply:
I'm fine with that, and changed it to `getAdapterInfo()` in the PR.

If it is going to be a method, I'm tempted to condense both functions into one, have it always return a promise, rename `hints` as `unmaskHints`, and indicate in the spec text that requesting any `unmaskHints` may incur user confirmation. I recall someone in a previous review (Dominic?) saying that it was a little weird to have methods that ask for user consent with some inputs but not others, but it's also not unheard of. (WebXR does it at the very least, and I'm pretty sure I found some prior art before specifying that.)

Reply:
After talking about it briefly with Kai, the approach I outlined actually feels bette overall so I've updated the PR to reflect it.